### PR TITLE
Swope adv app new

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: applications/config_helm_watch.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
+
+[#helm-watch-config]
+= Configuring Helm to watch namespace resources
+
+By default, when a subscription deploys subscribed Helm applications to target clusters, the application resources are watched. You can configure the Helm application to watch namespace-scoped resources.
+
+//WHY DO YOU WANT TO DO THIS?
```
Comment (mikeshng) [inline applications/config_helm_watch.adoc]: When enabled, manual changes to watched namespace-scoped resources will be reverted.

File: applications/config_helm_watch.adoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+[#prerequisites-argo]
+== Prerequisites 
+
+//need this info, what user needs before
```
Comment (mikeshng) [inline applications/config_helm_watch.adoc]: This option is only available for Helm Channel type.

File: applications/channel_sample.adoc [hunk 3]
```diff
@@ -55,45 +55,56 @@ spec:
...
 
 | metadata.name
-| Required. The name of the channel.
+| Required
+| The name of the channel.
 
 | metadata.namespace
-| Required. The namespace for the channel; Each channel needs a unique namespace, except the Git channel.
+| Required
+| The namespace for the channel; Each channel needs a unique namespace, except the Git channel.
```
Comment (dockerymick) [inline applications/channel_sample.adoc:74]: I would change the semicolon to a period.

File: applications/channel_sample.adoc [hunk 4]
```diff
@@ -55,45 +55,56 @@ spec:
...
 
 | spec.sourceNamespaces
-| Optional. Identifies the namespace that the channel controller monitors for new or updated deployables to retrieve and promote to the channel.
+| Optional
+| Identifies the namespace that the channel controller monitors for new or updated deployables to retrieve and promote to the channel.
 
 | spec.type
-| Required. The channel type. The supported types are: `HelmRepo`, `Git`, and `ObjectBucket` (Object storage in the console)
+| Required
+| The channel type. The supported types are: `HelmRepo`, `Git`, and `ObjectBucket` (Object storage in the console)
```
Comment (dockerymick) [inline applications/channel_sample.adoc:82]: add a period at the end

File: applications/channel_sample.adoc [hunk 5]
```diff
@@ -55,45 +55,56 @@ spec:
...
 | spec.pathname
-| Required for `HelmRepo`, `Git`, `ObjectBucket` channels. For a `HelmRepo` channel, set the value to be the URL for the Helm repository. For an `ObjectBucket` channel, set the value to be the URL for the Object storage. For a `Git` channel, set the value to be the HTTPS URL for the Git repository.
+| Required for `HelmRepo`, `Git`, `ObjectBucket` channels 
+| For a `HelmRepo` channel, set the value to be the URL for the Helm repository. For an `ObjectBucket` channel, set the value to be the URL for the Object storage. For a `Git` channel, set the value to be the HTTPS URL for the Git repository.
 
 | spec.secretRef.name
-| Optional. Identifies a Kubernetes Secret resource to use for authentication, such as for accessing a repository or chart.
+| Optional
+| Identifies a Kubernetes Secret resource to use for authentication, such as for accessing a repository or chart.
 You can use a secret for authentication with only `HelmRepo`, `ObjectBucket`, and `Git` type channels.
```
Comment (dockerymick) [inline applications/channel_sample.adoc:92]: Are you looking to create a break line for the statement on line 91? If yes, add one more break, if not you can move the sentence from 91 to 90

File: applications/channel_sample.adoc [hunk 6]
```diff
@@ -55,45 +55,56 @@ spec:
...
 
 | spec.gates
-| Optional. Defines requirements for promoting a deployable within the channel. If no requirements are set, any deployable that is added to the channel namespace or source is promoted to the channel. The `gates` value is only for `ObjectBucket` channel types and does not apply to `HelmRepo` and `Git` channel types, .
+| Optional
+| Defines requirements for promoting a deployable within the channel. If no requirements are set, any deployable that is added to the channel namespace or source is promoted to the channel. The `gates` value is only for `ObjectBucket` channel types and does not apply to `HelmRepo` and `Git` channel types, .
```
Comment (dockerymick) [inline applications/channel_sample.adoc:95]: extra space and comma here

File: applications/channel_sample.adoc [hunk 7]
```diff
@@ -55,45 +55,56 @@ spec:
...
 | metadata.labels
-| Optional. The labels for the channel.
+| Optional
+| The labels for the channel.
 
 | spec.insecureSkipVerify 
-| Optional. Default value is `false`, if set `true`, the channel connection is built by skipping the authentication
-
+| Optional
+| Default value is `false`, if set `true`, the channel connection is built by skipping the authentication
```
Comment (dockerymick) [inline applications/channel_sample.adoc:107]: add period

File: applications/placement_sample.adoc [hunk 8]
```diff
@@ -61,62 +61,79 @@ spec:
...
 
 | metadata.labels
-| Optional. The labels for the placement rule.
+| Optional
+| The labels for the placement rule.
 
 | spec.clusterSelector
-| Optional. The labels for identifying the target clusters
+| Optional
+| The labels for identifying the target clusters
```
Comment (dockerymick) [inline applications/placement_sample.adoc:92]: add a period

File: applications/placement_sample.adoc [hunk 9]
```diff
@@ -61,62 +61,79 @@ spec:
...
 
 | spec.clusterSelector.matchLabels
-| Optional. The labels that must exist for the target clusters.
+| Optional
+| The labels that must exist for the target clusters.
 
 | spec.clusterSelector.matchExpressions
-| Optional. The labels that must exist for the target clusters.
+| Optional
+| The labels that must exist for the target clusters.
```
Comment (dockerymick) [inline applications/placement_sample.adoc:100]: Should this be the match expressions that must exist for target clusters?

File: applications/placement_sample.adoc [hunk 10]
```diff
@@ -61,62 +61,79 @@ spec:
...
 
 | status.decisions
-| Optional. Defines the target clusters where deployables are placed.
+| Optional
+| Defines the target clusters where deployables are placed.
 
 | status.decisions.clusterName
-| Optional. The name of a target cluster
+| Optional
+| The name of a target cluster
```
Comment (dockerymick) [inline applications/placement_sample.adoc:108]: add a period

File: applications/channel_sample.adoc [hunk 11]
```diff
@@ -55,45 +55,56 @@ spec:
 == Channel YAML table
 
 |===
-| Field | Description
+| Field | Required or Optional| Description
```
Comment (dockerymick) [inline applications/channel_sample.adoc]: I have Optional or required

Is there a reason why we want required first? I initially just put optional first bc of alphabetical order haha

File: applications/subscription_sample.adoc [hunk 12]
```diff
@@ -79,113 +79,152 @@ spec:
...
 | spec.placement.clusterSelector
-| Optional. Defines the label selector to use to identify the clusters where the subscription is to be placed.
-Use only one of `clusters`, `clusterSelector`, or `placementRef` to define where your subscription is to be placed for a multicluster. If your cluster is a stand-alone cluster that is not your hub cluster, you can also use `local cluster`.
+| Optional
+| Defines the label selector to use to identify the clusters where the subscription is to be placed. Use only one of `clusters`, `clusterSelector`, or `placementRef` to define where your subscription is to be placed for a multicluster. If your cluster is a stand-alone cluster that is not your hub cluster, you can also use `local cluster`.
 
 | spec.placement.placementRef
-| Optional. Defines the placement rule to use for the subscription.
+| Optional
+| Defines the placement rule to use for the subscription.
```
Comment (dockerymick) [inline applications/subscription_sample.adoc:178]: i would move the statement from 179 to 178, unless you want to add a break line

File: applications/subscription_sample.adoc [hunk 13]
```diff
@@ -79,113 +79,152 @@ spec:
...
 Use only one of `clusters`, `clusterSelector` , or `placementRef` to define where your subscription is to be placed for a multicluster. If your cluster is a stand-alone cluster that is not your Hub cluster, you can also use `local cluster`.
 
 | spec.placement.placementRef.name
-| Optional, but required for using a placement rule. The name of the placement rule for the subscription.
+| Optional, but required for using a placement rule
+| The name of the placement rule for the subscription.
 
 | spec.placement.placementRef.kind
-| Optional, but required for using a placement rule. Set the value to `PlacementRule` to indicate that a placement rule is used for deployments with the subscription.
+| Optional, but required for using a placement rule.
```
Comment (dockerymick) [inline applications/subscription_sample.adoc:186]: I recommend removing the period here for consistency. In the other rows and tables, you are not using a period

**Review by dockerymick — APPROVED** [review]
Provided my feedback for the tables. It seems like you are still applying changes after dev review. let me know if you need another review. Otherwise, this looks good to me

File: applications/channel_sample.adoc (hunk 11 above)
Comment (swopebe) [reply applications/channel_sample.adoc]: I will follow yours.

File: applications/channel_sample.adoc (hunk 3 above)
Comment (swopebe) [reply applications/channel_sample.adoc:74]: Not a complete sentence, need to rework...this PR didn't change the content, only the table.

File: applications/channel_sample.adoc (hunk 7 above)
Comment (swopebe) [reply applications/channel_sample.adoc:107]: It's not really a complete sentence, so I don't know if a period is accurate...will rework.

File: applications/placement_sample.adoc (hunk 9 above)
Comment (swopebe) [reply applications/placement_sample.adoc:100]: also not a complete sentence...

on all of these I didn't work the content, that is legacy. So some of them are not accurately punctuated. Will revisit.

**Comment by mikeshng** [comment]
I've reviewed the Helm channel type related changes and they look good. Thanks.

**Pull request merged**